### ISSUE TITLE
ESLint with Google style warning on properly formatted code

### ISSUE THREAD

**Issue by janakdr** [eslint/eslint#12083]
The following code is unchanged by `clang-format --style=Google`:

```javascript
/**
 * @param {number} param1
 * @param {number} param2
 * @return {number}
 */
function myFunction(param1, param2) {
  const aVeryLongArgumentThatDoesNotFitAsAParameter = 0;
  const resultOfCallingFunction =
      myFunction(myFunction(
                    aVeryLongArgumentThatDoesNotFitAsAParameter,
                    aVeryLongArgumentThatDoesNotFitAsAParameter))
          .value;
  return resultOfCallingFunction;
}

myFunction(0, 0);
```

However, running it with ESLint gives the errors:

```
test.js
  10:1  error  Expected indentation of 10 spaces but found 20  indent
  11:1  error  Expected indentation of 10 spaces but found 20  indent
```

ESLint apparently wants the lines above to be:

```javascript
    myFunction(myFunction(
        aVeryLongArgumentThatDoesNotFitAsAParameter,
        aVeryLongArgumentThatDoesNotFitAsAParameter))
```
This seems inconsistent with clang-format's concept of ContinuationIndentWidth. Is there a workaround for this? Or is my setup wrong? The contents of my .eslintrc are:

```
module.exports = {
  'env': {
    'browser': true,
    'es6': true,
  },
  'extends': [
    'google',
  ],
  'globals': {
    'Atomics': 'readonly',
    'SharedArrayBuffer': 'readonly',
  },
  'parserOptions': {
    'ecmaVersion': 2018,
    'sourceType': 'module',
  },
  'rules': {
  },
};
```

**Comment by kaicataldo** [comment]
It looks like you're extending off of [`eslint-config-google`](https://github.com/google/eslint-config-google). The configuration for that project can be found [here](https://github.com/google/eslint-config-google/blob/master/index.js#L208-L228). I'm not sure if there's a way to enforce this with the core indentation rule or not, but if you feel that `eslint-config-google` is not enforcing what you expect, it's probably best to make an issue there.

**Comment by janakdr** [comment]
Thanks! Filed https://github.com/google/eslint-config-google/issues/58
